### ISSUE TITLE
Diodeless version

### ISSUE THREAD

**Issue by joostwestra** [soundmonster/samoklava#17]
Maybe "issue" is bit of a strong term here.
But since this is a purely wireless focussed build. Would it not be better to generate it as a wireless version. With 36 keys and no trrs connection you have exactly the right number of pins to use a diodeless layout.

**Comment by Shahabaz-Bagwan** [comment]
https://github.com/andrewjrae/rae-dux This is what you are looking for

**Comment by joostwestra** [comment]
Yes I found that branch as well and had a chat with the branch owner. Unfortunately the auto routing does not finish on his board.

**Comment by stuart-warren** [comment]
I'd like the same thing.

This one might work, I haven't actually ordered/tested yet
https://github.com/stuart-warren/samoklava/tree/36sweep

I'm not sure why the images are out of date though
